src/pool/bridgePool.e2e.ts

```diff
@@ -0,0 +1,149 @@
+import dotenv from "dotenv";
+import * as uma from "@uma/sdk";
+import { Client, Provider, PoolEventState } from "./bridgePool";
+import { ethers, BigNumber } from "ethers";
+import { TransactionReceipt } from "@ethersproject/abstract-provider";
+import assert from "assert";
+import set from "lodash/set";
+import get from "lodash/get";
+
+const { bridgePool } = uma.clients;
+
+dotenv.config();
```

Comment:
I'm just thinking, if each "sub-package" will require a different set of env variables, would be necessary to have one .env file for each 'sub-package' or keep a single .env file for the whole project?

Reply:
hmm, good question, i would just keep a single env but open to a different way